### ISSUE TITLE
vim.treesitter.query.get_query is deprecated

### ISSUE THREAD

**Issue by asyncedd** [andymass/vim-matchup#279]
### Explain the issue
I get a deprecation note on starting the plugin.

Most issues are related to bugs or problems. In these cases, you should
include a minimal working example and a minimal vimrc file (see below), as
well as:

1. What vim version are you using?  If using neovim, Are you using
   tree-sitter or classic matching?
```
NVIM v0.9.0-dev-1277+gaab4443bb
Build type: Release
LuaJIT 2.1.0-beta3

   system vimrc file: "$VIM/sysinit.vim"
  fall-back for $VIM: "/usr/local/share/nvim"

Run :checkhealth for more info
```

Occurs on treesitter. (via nvim-treesitter)

2. Steps to reproduce
Have nvim-treesitter installed.
Enable highlighting and install the Lua parser.
Also have matchup.
3. Expected behavior
working as intended, no errors.
5. Observed behavior
Gets error when starting the plugin.
```
vim.treesitter.query.get_query() is deprecated, use vim.treesitter.query.get() instead. :help deprecated                                                                                                            
This feature will be removed in Nvim version 0.10                                                                                                                                                                   
stack traceback:                                                                                                                                                                                                    
        /usr/local/share/nvim/runtime/lua/vim/treesitter/query.lua:200: in function 'get_query'                                                                                                                     
        ...vim-matchup/lua/treesitter-matchup/third-party/query.lua:55: in function 'get_query'                                                                                                                     
        ...vim-matchup/lua/treesitter-matchup/third-party/query.lua:142: in function 'prepare_query'                                                                                                                
        ...vim-matchup/lua/treesitter-matchup/third-party/query.lua:339: in function 'iter_group_results'                                                                                                           
        ...vim-matchup/lua/treesitter-matchup/third-party/query.lua:351: in function 'collect_group_results'                                                                                                        
        ...vim-matchup/lua/treesitter-matchup/third-party/query.lua:23: in function 'update_cached_matches'                                                                                                         
        ...vim-matchup/lua/treesitter-matchup/third-party/query.lua:31: in function 'get_matches'                                                                                                                   
        .../nvim/lazy/vim-matchup/lua/treesitter-matchup/syntax.lua:26: in function 'get_skips'                                                                                                                     
        .../nvim/lazy/vim-matchup/lua/treesitter-matchup/syntax.lua:60: in function 'lang_skip'                                                                                                                     
        [string "luaeval()"]:1: in main chunk
```

If your issue is instead a feature request or anything else, please
consider if minimal examples and vimrc files might still be relevant.


### Minimal working example

Please provide a minimal working example, e.g.,

This occurs on startup.

### Minimal vimrc file

Please provide a minimal vimrc file that reproduces the issue. The
following should often suffice:

```lua
local root = vim.fn.fnamemodify("./.repro", ":p")

-- set stdpaths to use .repro
for _, name in ipairs({ "config", "data", "state", "cache" }) do
  vim.env[("XDG_%s_HOME"):format(name:upper())] = root .. "/" .. name
end

-- bootstrap lazy
local lazypath = root .. "/plugins/lazy.nvim"
if not vim.loop.fs_stat(lazypath) then
  vim.fn.system({
    "git",
    "clone",
    "--filter=blob:none",
    "--single-branch",
    "https://github.com/folke/lazy.nvim.git",
    lazypath,
  })
end
vim.opt.runtimepath:prepend(lazypath)

-- install plugins
local plugins = {
  -- do not remove the colorscheme!
  "folke/tokyonight.nvim",
  {
    "andymass/vim-matchup",
    config = function()
      vim.g.matchup_matchparen_offscreen = { method = "popup" }
    end,
  },
  {
    "nvim-treesitter/nvim-treesitter",
    config = function()
      require("nvim-treesitter.configs").setup({
	highlight = { enable = true },
	matchup = { enable = true },
	ensure_installed = { "lua" },
      })
    end,
  },
  -- add any other pugins here
}
require("lazy").setup(plugins, {
  root = root .. "/plugins",
})

-- add anything else here
vim.opt.termguicolors = true
-- do not remove the colorscheme!
vim.cmd([[colorscheme tokyonight]])
```

**Comment by andymass** [comment]
Is this fixed after #278?

**Comment by asyncedd** [comment]
Yes, thank you!
